src/patterns/telephone-numbers/index.md.njk

```diff
@@ -76,11 +76,14 @@ It’s possible to mark up telephone numbers as links, like this:
 <a href="tel:+442079476330">020 7947 6330</a>
 ```
 
-However, doing this will style telephone numbers as links, which is confusing on devices that do not support telephone calls, like most desktop machines.
+However, doing this will style telephone numbers as links, which is confusing on devices that do not support telephone calls.
 
-It’s also not necessary - most modern mobile browsers automatically detect telephone numbers and display them as links anyway.
+It might also not be necessary - some modern mobile browsers automatically detect telephone numbers and display them as links anyway.
+
+If you do need to mark up your telephone numbers as links, for example, to support a device that cannot automatically detect them, make sure they do not display as links on devices that cannot make calls.
+
+If the user opens the link on a telephone number, it may trigger a setup process to install a communications app. You should consider whether this behaviour is part of the user journey you want to provide.
```

Comment:
On my first read, this para looked a tad patronising. It took me a while to get the context for this in the issue, but how about front-loading the 'user journey' message?

I'm also at a bit of a loss on how to be clear about the Skype situation without calling them out specifically. I think we try to avoid naming names in this 'layer' of guidance. 😄 

```suggestion
When looking at the user journey for your service, remember that telephone numbers as links might behave in unexpected ways for the user. For example, some browsers and operating systems will automatically start a setup process to install bundled software if the user has not set a default app to handle `tel:` links.
```

Reply:
@calvin-lau-sig7 That looks good, thanks! Might just add the _lightest_ of tweaks...

```suggestion
When you look at the user journey for your service, remember that telephone numbers as links might behave in unexpected ways for the user. For example, unless the user sets a default app to handle `tel:` links, some browsers and operating systems will automatically start a setup process.
```

Reply:
Looks good! Reading it, it feels more neutral and balance than what I'd wrote.

Approving now — feel free to commit and merge.